src/ApplicationCore/Entities/FraudProtectionApiModels/SignIn.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Microsoft.Dynamics.FraudProtection.Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Contoso.FraudProtection.ApplicationCore.Entities.FraudProtectionApiModels
+{
+    /// <summary>
+    /// Contains information and context about a sigin attempt.
```

Comment:
'sigin' typo ... also can you please make sure the comments consistently call it out?  e.g. I see all the following:

"Sign-in attempt"
"sign-in attempt" (I prefer this version)
"Sign-in ingestion"
"Sign-in event"